nibabel/freesurfer/io.py

```diff
@@ -388,6 +430,11 @@ def write_annot(filepath, labels, ctab, names):
         RGBA + label id colortable array.
     names : list of str
         The names of the labels. The length of the list is n_labels.
+    fill_ctab : bool
```

Comment:
Add `optional, as in 

```
fill_ctab : {True, False}, optional
```

The {True False} instead of bool also specifies the default.